assets/xml/overlays/ovl_Arrow_Fire.xml

```diff
@@ -0,0 +1,11 @@
+<Root>
+    <File Name="ovl_Arrow_Fire" BaseAddress="0x80865d10" RangeStart="0x9D0" RangeEnd="0x1DF0" Segment="128">    
+        <Texture Name="sFireArrow1Tex" OutName="fire_tex_1" Format="ia8" Width="32" Height="64" Offset="0x9D0"/>
+        <Texture Name="sFireArrow2Tex" OutName="fire_tex_2" Format="ia8" Width="32" Height="64" Offset="0x11D0"/>
+        <Array Name="sFireArrowVtx" Count="43" Offset="0x19D0">
+            <Vtx/>
+        </Array>
+        <DList Name="sFireArrowTextureDL" Offset="0x1C80"/>
+        <DList Name="sFireArrowVtxDL" Offset="0x1D30"/>
```

Comment:
sorry for bringing this up again, I still think this should use "Geometry", not "Vtx",
since the dlist both loads vertices and draws faces (triangles),
and vertex/edge/faces are commonly refered to as "geometry" in 3d modeling

Reply:
sFireArrowShapeDL or sFireArrowTriDL imo